jobs/e2e_runner.groovy

```diff
@@ -98,7 +98,8 @@ evaluate(new File("${workspace}/common.groovy"))
         make docker-test
 
         docker login -e="\$DOCKER_EMAIL" -u="\$DOCKER_USERNAME" -p="\$DOCKER_PASSWORD"
-        DEIS_REGISTRY='' make docker-build ${dockerPush}
+        # build once with "docker --pull --no-cache" to avoid stale layers
```

Comment:
No worries--if that gets merged first (nudge @bacongobbler), I'll clean this up.